src/net.rs

```diff
@@ -593,14 +593,17 @@ impl<S: SslClient> NetworkConnector for HttpsConnector<S> {
 }
 
 
-#[cfg(not(feature = "openssl"))]
+#[cfg(all(not(feature = "openssl"), not(feature = "security-framework")))]
 #[doc(hidden)]
 pub type DefaultConnector = HttpConnector;
 
 #[cfg(feature = "openssl")]
 #[doc(hidden)]
 pub type DefaultConnector = HttpsConnector<self::openssl::Openssl>;
 
+#[cfg(all(feature = "security-framework", not(feature = "openssl")))]
+pub type DefaultConnector = HttpsConnector<self::security_framework::ClientWrapper>;
```

Comment:
Openssl feature would still be enabled to provide the secure feature of cookies. So if you enable both, you'd be using the Openssl connector, right?